FLEDGE_implementation_overview.md

```diff
@@ -0,0 +1,66 @@
+
+So this timeout potentially includes executor process startup time, time to load bidding scripts and trusted bidding signals, time to wait for all interest groups to get bidding signals (if “enableBiddingSignalsPrioritization” is true for any of a bidder’s interest group), and, of course, time to generate a bid.  It does not include time waiting for Promises passed into the AuctionConfig to be resolved, time to load interest groups, or time to call scoreAd() for the bids a buyer generates.
+
+Note that if multiple auctions share a bidder process, regardless of whether they share executors or not, they share a Javascript thread, so there is potential contention for a shared resources, while both of their cumulative timeouts run down.  On the other hand, the bidder that starts second may be able to reuse an already created executor, so could avoid having the time to start a process and fetch the bidding script counting against the cumulative timeout.
+
+### Scoring Bids
+Once the seller executor has been loaded, of which there is always only one per AuctonConfig, and Promises resolved, bids are passed to scoreAd() as soon as they’re available, on a first-come-first-served basis.  If any additionalBids are passed in to an auction, they’ll likely be scored first.  Since ties are resolved randomly, and there’s currently no timeouts that span multiple scoreAd() calls, the order in which bids are scored currently does not matter.  Each scoreAd() call independently respects the scoreAd() timeout, which includes the time to run the Javascript file as well as the time to run scoreAd().
+
+Trusted seller signals fetches are started every 100 milliseconds, when there’s a bid a seller executor is waiting to score.  Once the final bid for an auction has been generated, the final signals fetch is immediately issued.  As with bidders, seller executors can be shared between auctions in the same frame running concurrently, which also allows for for shared seller signals fetches.
+
+Once all bids are scored, the result is immediately returned to the Javascript Promise.
+
+## Phase 3 - Reporting Scripts
+On auction completion, Chrome immediately reuses the seller executor to call reportResult(), and then reloads the winning bidder’s executor to call reportWin(), if it’s not still in memory.  The winning bidder executor is not kept in memory to avoid consuming bidder process quota, and to potentially avoid causing deadlock (e..g, if there are 10 auctions, all keeping different currently top-scoring bidders around, but that still need to load more bidders before they know the final winner, no auction could proceed).
```

Comment:
Doesn't a fenced frame have to navigate to the winning ad before reporting functions start running?